webserver/website/src/views/connectionDisplay.vue

```diff
@@ -126,4 +126,167 @@ export default {
 .connections-move {
   transition: transform 1s;
 }
+
+.custom_card {
```

Comment:
Solle man des hier in nen eigenes CSS file tun, oder lohnt sich des nicht?
Kp wie da so die conventions sind

Reply:
Die Convention ist, dass man für eine Website nur ein einziges riesiges css file hat. Finde ich aber nicht so gut. Immer da wo es zu den Components passt würde ich daher einfach alles in die .vue files machen (so wie es jetzt ist)